### PR TITLE
Remove deprecated pymongo calls in Chunkstore and Versionstore

### PR DESCRIPTION
**Pull request by bmoscon**

### CODE REVIEW

**Comment by bmoscon** [comment]
fixes #397

**Comment by kraphtuos** [comment]
Consider unpinning pymongo version in .travis.yml?

**Comment by bmoscon** [comment]
@kraphtuos thanks, I didnt know it got changed!

File: arctic/chunkstore/chunkstore.py [hunk 1]
```diff
@@ -360,16 +359,17 @@ def write(self, symbol, item, metadata=None, chunker=DateChunker(), audit=None,
                             START: start,
```
Comment (kraphtuos) [inline arctic/chunkstore/chunkstore.py]: line 357 (`op = True`) can be deleted
minor: eliminate `find` variable for consistency

Comment (bmoscon) [inline arctic/chunkstore/chunkstore.py]: fixed, thanks!

**Pull request merged**